apps/console/src/auth.html

```diff
@@ -258,12 +259,14 @@
                      * @returns {string} Contructed URL.
                      */
                     function signInRedirectURL() {
+                        var contextPath = proxyContextPathGlobal !== "null" ? "/" + proxyContextPathGlobal : "";
```

Comment:
what is the purpose of a proxy context path? just asking for edu. purposes 🙃

Reply:
You can define a `proxy_context_path` in IS and get the server hosted under a context path rather than the root.
With this, apps will have the following paths.

- `https://is.dev.wso2.com/auth/t/cloudstream.io/authenticationendpoint`
- `https://is.dev.wso2.com/auth/t/cloudstream.io/console`
- `https://is.dev.wso2.com/auth/t/cloudstream.io/myaccount`

```diff
[server]
hostname = "is.dev.wso2.com"
node_ip = "127.0.0.1"
base_path = "https://$ref{server.hostname}"
+ proxy_context_path = "auth"
```

Reply:
Ack, thanks 🙌